tests/integration-tests/conftest.py

```diff
@@ -369,7 +370,8 @@ def cfn_stacks_factory(request):
 AVAILABILITY_ZONE_OVERRIDES = {
     # c5.xlarge is not supported in us-east-1e
     # FSx Lustre file system creation is currently not supported for us-east-1e
-    "us-east-1": ["us-east-1a", "us-east-1b", "us-east-1c", "us-east-1d", "us-east-1f"],
+    # "us-east-1": ["us-east-1a", "us-east-1b", "us-east-1c", "us-east-1d", "us-east-1f"],
+    "us-east-1": ["us-east-1a", "us-east-1b"],
```

Comment:
I'm going to leave this in here as a reminder to somehow deal elsewhere in the integration tests with AZs that don't have ARM instances before merging into develop.